services/jobrunner/docker-compose.yaml

```diff
@@ -1,23 +1,27 @@
+x-base-service: &base-service
```

Comment:
What's the significance of `x-`?

Reply:
It's so that Docker Compose knows to ignore this stanza:
https://docs.docker.com/reference/compose-file/extension/